aerleon/lib/cisco.py

```diff
@@ -208,9 +208,10 @@ def __str__(self) -> str:
         # for using cisco, which has decided to implement its own meta language.
 
         # Create network object-groups
-        for name, ips in self.addressbook.addressbook[''].items():
+        for zone, name, ips, _ in self.addressbook.WalkAddressBook():
             for version in (4, 6):
                 vips = [i for i in ips if i.version == version]
+                vips = nacaddr.CollapseAddrList(vips)
```

Comment:
Is the addition of CollapseAddrList here related to the FQDN / AddressBook change?